sql/core/src/test/scala/org/apache/spark/sql/execution/TakeOrderedAndProjectSuite.scala

```diff
@@ -93,4 +93,38 @@ class TakeOrderedAndProjectSuite extends SparkPlanTest with SharedSparkSession {
       }
     }
   }
+
+  test("TakeOrderedAndProject.doExecute with local sort") {
```

Comment:
Hmm, isn't this test already passed without the change?

Reply:
It should pass since this pr is not a bug fix. This test is for ensure the data correctness after improve the related code.

Reply:
I see. Makes sense.